src/components/pages/devices/devices.js

```diff
@@ -1,14 +1,30 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
+import { Btn } from 'components/shared';
 
 import './devices.css';
 
 export class Devices extends Component {
-  render () {
+  constructor(props) {
+    super(props);
+    if (props.initialize) {
+      this.props.initialize();
+    }
+  }
+
+  render() {
+    const { t, devices, refresh } = this.props;
+    var listItems = t('components.devices.noneFound');
```

Comment:
Avoid using `var`. Default to using `const` and `let` when `const` is unsuitable. More about this below.